notificationservice/src/main/java/com/example/notificationservice/models/responsebody/BlacklistResponseBody.java

```diff
@@ -0,0 +1,33 @@
+package com.example.notificationservice.models.responsebody;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.Data;
+import org.springframework.lang.Nullable;
+
+import java.util.List;
+
+@Builder
+@Data
+public class BlacklistResponseBody {
```

Comment:
Instead of writing @Nullable for each field, you can have these two at the top of the class. They will ignore if any null fields are there and will deserialize only non null fields. It will also ignore any other field which is coming and is not defined in this pojo
@JsonIgnoreProperties(ignoreUnknown = true)
@JsonInclude(JsonInclude.Include.NON_NULL)